PROPOSALS.md

```diff
@@ -131,6 +133,15 @@ the request, then it automatically passes.
 - for a simple proposal: whether it has been merged
 - for a multi-phase proposal: whether the next step has been carried out as planned
 
+### `base`: Bug fixes
+
+The CLC votes on concrete changes to `base` and not designs. If a change to `base`
```

Comment:
This is good to have as a section, but I'd say the sentence "The CLC votes on concrete changes to base and not just designs" should be more prominent and moved to the [base package](https://github.com/haskell/core-libraries-committee?tab=readme-ov-file#base-package) section of the main README.

I've seen it recently that people open issues not being aware that they're expected to implement the entire proposal themselves.